web/packages/extension/build/_locales/sv/messages.json

```diff
@@ -0,0 +1,56 @@
+{
+    "settings_ruffle_enable": {
+        "message": "Spela upp Flash-innehåll i Ruffle"
+    },
+    "settings_page_ignore_optout": {
+        "message": "Spela upp Flash-innehåll även på sidor som inaktiverar Ruffle"
+    },
+    "settings_ignore_optout": {
+        "message": "Ignorera webbsidekompatibilitetsvarningar"
```

Comment:
Is there really no better wording than "webbsidekompatibilitetsvarningar"? I can't think of anything that'd be much better but this looks a bit silly.

Reply:
Yeah I was unsure about this one as well. Felt like I was writing German. I can't think of anything better either myself however.